k8s/fluentd-s3-scheduler.yml

```diff
@@ -0,0 +1,89 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: fluentd-s3-scheduler
+  namespace: classroom
+automountServiceAccountToken: true
+
+
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: fluentd-s3-scheduler
+  namespace: classroom
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+subjects:
+- kind: ServiceAccount
+  name: fluentd-s3-scheduler
+  namespace: classroom
+
+
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  namespace: classroom
+  name: fluentd-s3-scheduler
+spec:
+  schedule: "*/5 * * * *"
+  concurrencyPolicy: Replace
+  successfulJobsHistoryLimit: 2
+  failedJobsHistoryLimit: 5
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          serviceAccountName: fluentd-s3-scheduler
+          automountServiceAccountToken: true
+          containers:
+          - name: scheduler
+            image: wildflowerschools/k8s-task-scheduler:v6
+            env:
+            - name: CONFIG_PATH
+              value: "/app/config/schedule.yaml"
+            - name: K8S_CLUSTER_NAME
+              value: "wework"
+            volumeMounts:
+            - name: config-volume
+              mountPath: /app/config
+          restartPolicy: OnFailure
+          volumes:
+          - name: config-volume
+            configMap:
+              name: fluentd-s3-schedule-config
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  creationTimestamp: 2016-02-18T18:52:05Z
+  name: fluentd-s3-schedule-config
+  namespace: classroom
+data:
+  schedule.yaml: |
+    timezone: "US/Central"
```

Comment:
@optimuspaul  -  Calling out this todo. As we deploy to more classrooms, I imagine the TZ will need to be configurable per deployment.

Reply:
yes, this will be the template for something that we can inject variables into per classroom in the future.